examples/go-client/access_workflow.go

```diff
@@ -44,7 +45,7 @@ func accessWorkflow(ctx context.Context, client *auth.Client) {
 	}()
 
 	// retrieve all pending access requests
-	filter := services.AccessRequestFilter{State: services.RequestState_PENDING}
+	filter := services.AccessRequestFilter{State: types.RequestState_PENDING}
```

Comment:
Do the docs pull from this file? If so I'd leave it `services` for now until you're fully done. If someone deploys from master, the docs will be wrong.